cve_bin_tool/output_engine/html_reports/js/main.js

```diff
@@ -1,40 +1,112 @@
 function searchProductsScanned() {
-    let input = document.getElementById('searchInput');
-    let filter = input.value.toLowerCase();
-    let ul = document.getElementById('listProducts');
-    let li = ul.getElementsByClassName('list-group-item-action')
-
-    for (let i = 0; i < li.length; i++) {
-        let txtValue = li[i].textContent || li[i].innerText;
-        if (txtValue.toLowerCase().indexOf(filter) > -1) {
-            li[i].style.display = "";
-        } else {
-            li[i].style.display = "none";
-        }
+  let input = document.getElementById('searchInput')
```

Comment:
Is there a particular reason to use two spaces rather than 4 in just this file?  I know it's javascript and not python but since our expected usage is local reports I'm not sure we care about saving a few bytes of space.  I'd rather be consistent across both for now unless there's a compelling reason.

Reply:
fixed! i didn't notice, prettifier replaced 4 spaces with 2 spaces.